build.cake

```diff
@@ -322,7 +320,7 @@ Task("DotnetPackServiceTools")
     {
         var outputFolder = System.IO.Path.Combine(nugetPackageFolder);
         var projectFolder = System.IO.Path.Combine(sourceFolder, project);
-        DotnetPack(outputFolder, projectFolder, project);
```

Comment:
Would suggest updating task name to `DotnetPackServiceToolsNoBuild`

Reply:
I feel updating task name to add "NoBuild" isn't necessary. The other task names don't have this much information in their name either. Let me know if you still strongly feel that this should be updated.

Reply:
Either way is fine, but generally I find having names be more explicit isn't a bad thing. Not a blocker though.